pkg/controller/user-cluster-controller-manager/resources/controller.go

```diff
@@ -20,6 +20,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	types2 "github.com/kubermatic/machine-controller/pkg/providerconfig/types"
```

Comment:
* Please make sure the import order is correct, i.e. this import should go along with other Kubermatic imports.
* Please give it a more descriptive import alias, such as `mctypes` or something like that.

Reply:
Try https://github.com/xrstf/gimps ^^